onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -64,8 +64,8 @@ def optimize_by_onnxruntime(
     use_gpu: bool = False,
     optimized_model_path: Optional[str] = None,
     opt_level: Optional[int] = 99,
-    disabled_optimizers=[],  # noqa: B006
-    verbose=False,
+    disabled_optimizers: List[str] = [],  # noqa: B006
+    verbose: bool = False,
```

Comment:
I suggest to add some optional parameters
 save_as_external_data : bool = False,
 external_data_filename: str = "",
 external_data_threshold: int = 1024

Reply:
Added